app/utils/helpers.rb

```diff
@@ -7,7 +7,7 @@ def to_tag(s)
 
 def is_time?(t)
```

Comment:
originally it didn't check for `nil`.
I had to modify it later because `nil` was also an acceptable value, but left the function name as is so I won't need to do lots of changes in the code.